docs/disconnected-cincinnati-operator.md

```diff
@@ -240,6 +240,8 @@ You might want to review the documentation around disconnected registries to lea
 
     gives a full URI.
 
+    If you see `host name validation error` in your UpdateService's status, try creating an UpdateService with a shorter name. 
```

Comment:
s/UpdateService/OpenShift update service/. Also we should exactly mention the limit i.e. 63 characters.